packages/components/src/spectrum/picker/Picker.tsx

```diff
@@ -71,10 +77,25 @@ export function Picker({
     [tooltip]
   );
 
+  const renderItem = useCallback(
+    ({ content, textValue }: NormalizedPickerItem) => (
+      <Item textValue={textValue === '' ? 'Empty' : textValue}>
```

Comment:
Was this `'Empty'` string part of the spec? Seems odd to have, as I'm not seeing anything else to visually distinguish this now empty string from a value of `Empty`.
In addition, we should be passing the `key` into `Item` here - that seems like a bug that we're not.

Reply:
`textValue` is an accessibility thing that isn't visible to the user. It shows warnings if set to empty string. 

`Item` is not actually rendered, and the `key` gets auto determined from the items in the `items` prop. That said, it shouldn't hurt to be explicit, so I can add.

Reply:
@mofojed  More details on how the `textValue` prop is used. It seems to get used as part of `<select><option>` elements that back the Picker. Specifically it seems to provide the content of the `<option>` elements, while the `value` attribute comes from the `key`

<img width="265" alt="image" src="https://github.com/deephaven/web-client-ui/assets/1900643/eaf7c543-1be7-4ccb-85fa-02b80ad858a4">

Reply:
Ah my concern was that this would fire with `Empty` on the change event. But if it's firing the event key that's okay. Disregard my comment.